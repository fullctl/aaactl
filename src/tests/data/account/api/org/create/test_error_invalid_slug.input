{
  "name": "Foo Bar Org",
  "slug": "api"
}
