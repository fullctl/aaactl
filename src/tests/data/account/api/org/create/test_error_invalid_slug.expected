{"data": [], "errors": {"slug": ["This name is reserved and may not be used"]}, "profiling": {"queries": 0}}
